### DISCUSSION TITLE
Product key is missing from sub.streetcare.us or domain name is incorrect

### DISCUSSION THREAD

**Discussion by 687000** [dalenguyen/firebase-wordpress-plugin#207] [Q&A] [answered]
Hi, when using the pro version, I could not use the shortcode or create/request data from the firebase.

When using the shortcode, I got this error message:
"[Firebase] - product key is missing from sub.streetcare.us or domain name is incorrect (missing or having extra www). Please visit https://techcater.com/shop/my-account to update your configuration."

When trying to create a new user, I got this error message:
"Credential implementation provided to initializeApp() via the "credential" property has insufficient permission to access the requested resource. See https://firebase.google.com/docs/admin/setup for details on how to authenticate this SDK with appropriate permissions."

May I ask what domain name should I provide and where I can update this configuration? I have already created and connected with the Firebase cloud functions and filled the settings. And the firebase credentials are copied exactly the same from firebase project app.

**Comment by dalenguyen** [comment, marked as answer]
Hi @687000, 

In this case, you should add `sub.streetcare.us` to https://techcater.com/shop/my-account page under the product key section. Please wait about an hour or change browser, the error will disappear.

Please take a snapshot if there is other error.

Thanks,
